### PR TITLE
Add e2e test for loading journal storage in standalone app

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

To follow-up on https://github.com/optuna/optuna-dashboard/pull/732

## What does this implement/fix? Explain your changes.

Add e2e test for loading journal storage in standalone app

### CODE REVIEW

**Comment by c-bata** [comment]
@gen740 @nabenabe0928 Could you review this PR?

**Review by gen740 — CHANGES_REQUESTED** [review]
Thank you for your PR!
I have a question regarding `test_load_rdb_storage` and `test_load_journal_storage`. They share the same code and test the same functionality. How about using a pytest fixture?

**Comment by keisuke-umezawa** [comment]
@gen740 
I updated it. Could you check it again?

File: e2e_tests/test_standalone/test_study_list.py [hunk 1]
```diff
@@ -25,31 +28,49 @@ def test_home(
...
+        x2 = trial.suggest_float("x2", 0, 10)
+        return (x1 - 2) ** 2 + (x2 - 5) ** 2
+
+    study.optimize(objective, n_trials=100)
+
+
+@pytest.mark.parametrize(
+    "create_storage_file",
+    [create_rdb_storage_sqlite_file, create_journal_storage_local_file],
+)
```
Comment (nabenabe0928) [inline e2e_tests/test_standalone/test_study_list.py:60]: [suggestion] How about bundling them together?
```suggestion
def create_rdb_storage_file(filename: str, study_name: str, storage_type: str):
    if storage_type == "rdb":
        storage = optuna.storages.RDBStorage(f"sqlite:///{filename}")
    elif storage_type == "journal":
        storage = optuna.storages.JournalStorage(
            optuna.storages.JournalFileStorage(f"{filename}"),
        )
    else:
        assert False, f"Got an unexpected storage_type={storage_type}."

    study = optuna.create_study(study_name=study_name, storage=storage)

    def objective(trial: optuna.Trial) -> float:
        x1 = trial.suggest_float("x1", 0, 10)
        x2 = trial.suggest_float("x2", 0, 10)
        return (x1 - 2) ** 2 + (x2 - 5) ** 2

    study.optimize(objective, n_trials=100)


@pytest.mark.parametrize("storage_type", ["rdb", "journal"])
```

**Review by nabenabe0928 — COMMENTED** [review]
Thank you for the PR and sorry for the late response, I left minor suggestions!

**Comment by keisuke-umezawa** [comment]
@nabenabe0928 @gen740 I fixed it. Could you review it again?

**Review by gen740 — APPROVED** [review]
Sorry for the late reply. LGTM!

**Review by nabenabe0928 — APPROVED** [review]
Thank you for the changes, LGTM!

**Pull request merged**